### ISSUE TITLE
new hosting

### ISSUE THREAD

**Issue by juliangruber** [juliangruber/ghub.io#17]
for various reasons i can't host this any more. @rvagg, @mathiasbynens, @Raynos or anyone else, would you be willing to help out?

this page has ~2k hits a month and could probably show some ads on its help page.

**Comment by rvagg** [comment]
yeah, I could stick it on the same server(s) as nodei.co pretty easily.

**Comment by juliangruber** [comment]
awesome! twitter dm'ed you

**Comment by terinjokes** [comment]
@rvagg i see this is now on the nodei.co server. any chance you can get a valid SSL certificate for it setup? Let's Encrypt, CloudFlare?

**Comment by bfred-it** [comment]
+1 for HTTPS support

**Comment by zeke** [comment]
I just created [package.land](https://package.land), which does exactly the same thing as ghub.io, but with a different implementation. I'm working on automating daily publishes of the [all-the-package-repos](https://github.com/nice-registry/all-the-package-repos) module on which package.land depends, and plan to enable greenkeeper on the repo to keep the repo list reasonably up to date as new modules are published.

package.land is pretty darn fast. It's running on a Heroku Hobby dyno for $7/month, and I've got SSL set up on it. Pushes to the master branch of the repo are auto-deployed. I would be very happy to point the ghub.io domain at it and pay to keep it alive, if you all are up for that. I've been linking to ghub.io from all over the internet for several years now and maybe it's time to pay my dues :)

**Comment by juliangruber** [comment]
@zeke :heart:

I'm all for it

**Comment by zeke** [comment]
> I'm all for it

Sweet! Here's a rough plan. Tell me what you think:

- invite @juliangruber and anyone else to @nice-registry org
- rename nice-registry/package.land repo to nice-registry/ghub.io
- retire the old juliangruber/ghub.io with a notice in the README
- reimplement the CLI functionality function of the `ghub` module
- set up semantic-release to autopublish `ghub` changes to npm
- point ghub.io DNS to Heroku

I also think the analytics story is really interesting. Would be really cool if we publicized the package request stats somewhere. Maybe as a start we could just dump the request logs to a publicly readable S3 bucket or something..

**Comment by juliangruber** [comment]
+1

For stats, we can expose this in another iteration.

**Comment by juliangruber** [comment]
What does @rvagg think?

**Comment by rvagg** [comment]
fine by me! looks good @zeke

**Comment by zeke** [comment]
Yay!
